packages/babel-parser/test/fixtures/comments/basic/class-method-static-generator/input.js

```diff
@@ -0,0 +1,2 @@
+(class /* 1 */ { /* 2 */ static /* 3 */ * /* 4 */ f /* 5 */ (/* 6 */) /* 7 */ { /* 8 */ } /* 9 */ } /* 10 */);
```

Comment:
Currently Babel attaches comments `1` to `7` to the method's block statement (horribly). It is now fixed in this PR.